.github/workflows/DrainpipeDev.yml

```
@@ -8,43 +8,51 @@ on:
       - 'v*'
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  group: ${{ github.workflow }}-${{ github.ref_name || github.sha }}
```

```suggestion
  group: drainpipe-dev
```

---

We could just use `${{ github.workflow }}` in this case then, since that's a static string too.
```suggestion
  group: ${{ github.workflow }}
```